idx/cassandra/cassandra.go

```diff
@@ -575,9 +581,6 @@ func (c *CasIdx) addDefToArchive(def interning.MetricDefinitionInterned) error {
 	for attempts := 0; attempts < maxAttempts; attempts++ {
 		if attempts > 0 {
 			sleepTime := 100 * attempts
-			if sleepTime > 2000 {
```

Comment:
what's the reason for removing this?

Reply:
`maxAttempts := 5`
`for attempts := 0; attempts < maxAttempts; attempts++ {`
`sleepTime := 100 * attempts`
It's impossible for `sleepTime > 2000`